src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -5,6 +5,10 @@
 #include "GlobalAppearance.h"
 #include "GlobalAppearance.g.cpp"
 #include "GlobalAppearancePageNavigationState.g.cpp"
+
+#include <LibraryResources.h>

```

Comment:
This should be part of the precompiled header precomp.h or pch.h and not be necessary here

`<numeric>` should be in LibraryIncludes

Reply:
Are you sure putting it into `<LibraryIncludes>` or the `pch.h` file is a good idea?
`<numeric>` is basically only used in 2 places throughout the project.